### PR TITLE
add resources pricing calculator

### LINKED ISSUE #179: add pricing calculator in the dashboard
part of https://github.com/threefoldtech/home/issues/1286

### PR DESCRIPTION
**Pull request by omar-sherif9992**
### Related issues
- https://github.com/threefoldtech/tfgrid_dashboard/issues/179

### CODE REVIEW

File: src/explorer/components/Sidebar.vue [hunk 1]
```diff
@@ -56,6 +56,7 @@ export default class Sidebar extends Vue {
     { title: "Overview", icon: "mdi-monitor-dashboard", route: "/" },
     { title: "Nodes", icon: "mdi-resistor-nodes", route: "/nodes" },
     { title: "Farms", icon: "mdi-webpack", route: "/farms" },
+    { title: "Calculator", icon: "calculator", route: "/calculator" },
```
Comment (AhmedHanafy725) [inline src/explorer/components/Sidebar.vue]: why do we add the calculator in the explorer sidebar?

File: src/explorer/views/Calculator.vue [hunk 2]
```diff
@@ -0,0 +1,209 @@
+<template>
```
Comment (AhmedHanafy725) [inline src/explorer/views/Calculator.vue]: also, why is this file in the explorer?

File: src/explorer/components/Sidebar.vue (hunk 1 above)
Comment (zaelgohary) [reply src/explorer/components/Sidebar.vue]: removed.

File: src/explorer/views/Calculator.vue (hunk 2 above)
Comment (zaelgohary) [reply src/explorer/views/Calculator.vue]: Was added earlier by mistake. removed now.

**Review by MohamedElmdary — APPROVED** [review]

**Pull request merged**